persons.txt

```diff
@@ -0,0 +1 @@
+[{"name":"Alex","height":108,"hair":"blue"},{"name":"Davi","height":108,"hair":"blue"}]
```

Comment:
This file should be added to gitignore because it's something that is used only in runtime.

The node_modules folder also should be added to gitignore.

Also where is package.json for this module?

Reply:
Changes made. Please check.

Reply:
Nice, but you have to delete this file from the repo now. Tip: remove from gitignore before committing the removal, then add again to gitignore.